src/main/java/org/gridsuite/securityanalysis/server/service/ActionsService.java

```diff
@@ -58,4 +58,21 @@ public List<ContingencyInfos> getContingencyList(String name, UUID networkUuid,
 
         return restTemplate.exchange(baseUri + path, HttpMethod.GET, null, new ParameterizedTypeReference<List<ContingencyInfos>>() { }).getBody();
     }
+
```

Comment:
should we remove: public List<ContingencyInfos> getContingencyList(String name, UUID networkUuid, String variantId) { 
as it's only used in test ?
And refactor the tests to use the new method ?

Reply:
done